lib/utils/constant.dart

```diff
@@ -1 +1,4 @@
 const double hundred = 100;
+const String imageURL = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/index.png';
+const int counterZero = 0;
```

Comment:
you just placed it in variable but still the same readability weight 😅 

how about 🙂 :
```
- const homeTabIndex = 0;
- const favoriteTabIndex = 0;
```